tests/Behat/Page/Admin/Invoice/ShowPage.php

```diff
@@ -138,7 +138,9 @@ public function hasTotal(string $total, string $currencyCode): bool
 
     public function getChannel(): string
     {
-        return $this->getElement('invoice_channel_name')->getText();
+        $items = $this->getDocument()->findAll('css', '.channel > .channel__item');
```

Comment:
Change the value of key `invoice_channel_name` and use it here

Reply:
This PR replaces channel widget with sylius one, which doesn't have `invoice_channel_name` id or any other ids.

https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminBundle/Resources/views/Common/_channel.html.twig

Though I don't understand which key you're mentioning.

Reply:
I'm talking about `invoice_channel_name` key that was used by `getElement` method before and it is configured below in `getDefinedElements` method. You should change the selector there and keep using `$this->getElement('invoice_channel_name')->getText();` if it possible

Reply:
Yes, it's not possible anymore to access the name via static path, so I will remove it from elements.

Implementation is pretty much the same as for RefundPlugin.
https://github.com/Sylius/RefundPlugin/blob/b263bf35834dc8f77a70d0d34ab9b5fc2fcc1453/tests/Behat/Page/Admin/CreditMemoDetailsPage.php#L78